### PR TITLE
IATIIMPORT-262 Implement Cookie manager, some refactoring and puting …

### PR DESCRIPTION
**Pull request by jdeanquin-dg**
…code in constant file

### CODE REVIEW

File: import-core/import-services/src/main/java/org/devgateway/importtool/services/ImportService.java [hunk 1]
```diff
@@ -153,15 +157,18 @@ public IDestinationProcessor getDestinationProcessor(String processorName, Strin
 				getName().equals(processorName)).findFirst();
 
 		if (optional.isPresent()) {
-			try {
-				Constructor<?> c = Class.forName(optional.get().getDestinationProcessor().getClassName()).getDeclaredConstructor(String.class);
-				c.setAccessible(true);
-				processor = (IDestinationProcessor) c.newInstance(new Object[]{authenticationToken});
-				processor.setProcessorVersion(processorVersion);
-			} catch (InstantiationException | IllegalAccessException | ClassNotFoundException | NoSuchMethodException | SecurityException | IllegalArgumentException | InvocationTargetException e) {
+			/*try {*/
```
Comment (fferreyra) [inline import-core/import-services/src/main/java/org/devgateway/importtool/services/ImportService.java]: Why is this commented out and not removed altogether?

File: import-core/import-services/src/main/java/org/devgateway/importtool/services/ImportService.java [hunk 2]
```diff
@@ -153,15 +157,18 @@ public IDestinationProcessor getDestinationProcessor(String processorName, Strin
...
+			processor = beanFactory.getBean(optional.get().getDestinationProcessor().getName() + "_PROCESSOR", IDestinationProcessor.class);
+			processor.setProcessorVersion(processorVersion);
+			processor.initialize(authenticationToken);
+			//Constructor<?> c = Class.forName(optional.get().getDestinationProcessor().getClassName()).getDeclaredConstructor(String.class);
```
Comment (fferreyra) [inline import-core/import-services/src/main/java/org/devgateway/importtool/services/ImportService.java]: same as previous

File: import-core/import-services/src/main/java/org/devgateway/importtool/services/ImportService.java (hunk 1 above)
Comment (jdeanquin-dg) [reply import-core/import-services/src/main/java/org/devgateway/importtool/services/ImportService.java]: Done!

File: import-core/import-services/src/main/java/org/devgateway/importtool/services/ImportService.java (hunk 2 above)
Comment (jdeanquin-dg) [reply import-core/import-services/src/main/java/org/devgateway/importtool/services/ImportService.java]: Done

**Pull request merged**